AdobeStockImage/etc/di.xml

```diff
@@ -8,4 +8,6 @@
 <config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:ObjectManager/etc/config.xsd">
     <preference for="Magento\AdobeStockImageApi\Api\Data\ImageInterface" type="Magento\AdobeStockImage\Model\Image"/>
     <preference for="Magento\AdobeStockImageApi\Api\GetImageListInterface" type="Magento\AdobeStockImage\Model\GetImageList"/>
+    <plugin name="adobe_stock_image_remove_metadata" type="Magento\AdobeStockImage\Plugin\Product\Gallery\Processor" sortOrder="10" disabled="false" />
```

Comment:
I believe that `plugin` node should be used inside of `type` node of the target class for a plugin

Reply:
Okay @sivaschenko, I think I misunderstood and remove the type node from this file on my last commit. It should be correct now.